File: src/main/java/com/wine/to/up/deployment/service/service/ApplicationInstanceManager.java
package com.wine.to.up.deployment.service.service;

import com.wine.to.up.deployment.service.vo.ApplicationInstanceVO;

public interface ApplicationInstanceManager {

    ApplicationInstanceVO startApplication(ApplicationInstanceVO applicationInstanceVO);

    ApplicationInstanceVO stopApplication(ApplicationInstanceVO applicationInstanceVO);

    ApplicationInstanceVO restartApplication(ApplicationInstanceVO applicationInstanceVO);
}

File: src/main/kotlin/com/wine/to/up/deployment/service/service/ApplicationInstanceManager.kt
package com.wine.to.up.deployment.service.service

import com.wine.to.up.deployment.service.vo.ApplicationInstanceVO

interface ApplicationInstanceManager {
    fun startApp( applicationInstanceVO: ApplicationInstanceVO)
    fun stopApp(applicationInstanceVO: ApplicationInstanceVO)
    fun restartApp(applicationInstanceVO: ApplicationInstanceVO)
}